packages/portal/src/components/item/ItemMediaThumbnail.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div
+    class="item-media-thumbnail text-lowercase"
+    data-qa="item media thumbnail"
+  >
+    <MediaCardImage
+      :media="resource"
+      :lazy="lazy"
+      :offset="offset"
+      :edm-type="edmType"
+      thumbnail-size="small"
+      :linkable="false"
+    />
+    <span class="thumbnail-page">{{ `p. ${offset + 1}` }}</span>
```

Comment:
Perhaps this should be a translation string, as I'm not sure the `p.` for page makes sense in all languages.